src/d3a/d3a_core/export.py

```diff
@@ -114,6 +115,17 @@ def _file_path(directory: dir, slug: str):
         file_name = ("%s.csv" % slug).replace(' ', '_')
         return directory.joinpath(file_name).as_posix()
 
+    def export_to_zip_file(self):
+        self.export()
```

Comment:
As discussed offline, what has to be decided is whether we want to export the htmls, too. They take a lot of space in the database and can be omitted IMO.

Reply:
Removed it already, discussion still pending.